README.md

```diff
@@ -49,33 +41,48 @@ The above step creates a local directory called ```cri-dockerd``` which you will
 
 To build this code (in a POSIX environment):
 
+```shell
```

Comment:
We should instead mention that Go needs to be installed and specify the current expected version

Reply:
There's directions for installing Go (however crudely) above; though we could have an explicit mention here and link back up to that anchor.

Reply:
I was getting at removing any install instructions for Go completely and instead mentioning that it is required and reference the official install docs

Reply:
Ah, I agree that would be a better direction. Alas, I despair that the users who tend to actually need the "strawman"/example install instructions we have will open issues asking how to install Go. We can just close those as out of scope of course; there's just an odd demographic coming here.

Reply:
Ah, now I see your concern. I would say we should definitely provide a link to https://go.dev/doc/install. I would hope that minimizes questions and gives them better support for installing on multiple OS :crossed_fingers:

Reply:
I did not change the Go instructions, I just moved them from the "install" section to the "build" (if any)
If users are downloading the precompiled tarball, they don't need to bother with setting up Go at all...

There are no hints for cross-compiling (GOOS=linux), so the instructions assume it is building on Linux.
So in that since I guess it was fair to have "installer_linux", even though I have never used that installer.

Reply:
Maybe the build step would also build for Windows, even if it is being run in a "POSIX environment" (MSYS?)

Reply:
> If users are downloading the precompiled tarball, they don't need to bother with setting up Go at all...

I think this is another reason that we shouldn't have instructions to install Go. Just point them to the official install docs and don't worry about keeping it up to date here. I'm not actually finding information on who maintains "installer_linux". It looks like there is a project called `getgo` to make an official installer but that's it.

> If there are better instructions in some other place, it would be possible to add a link to it in this repository README.

https://go.dev/doc/install